controller/users_blackbox_test.go

```diff
@@ -233,7 +238,7 @@ func (s *TestUsersSuite) TestUpdateUserUnsetVariableInContextInfo() {
 		"count":        3,
 	}
 
-	updateUsersPayload = createUpdateUsersPayload(&newEmail, &newFullName, &newBio, &newImageURL, &newProfileURL, contextInformation)
```

Comment:
> Do you mean to say, why did I pass nil for company *string instead of passing something real?

Yes, that is what I meant. I found it weird not to add it but that is all. 👍